core/trino-main/src/main/java/io/trino/execution/AddColumnTask.java

```diff
@@ -44,7 +44,8 @@
 import static io.trino.spi.StandardErrorCode.TABLE_NOT_FOUND;
```

Comment:
this should be squashed with commit that adds `SET PROPERTY` syntax

Reply:
I don't know which changes you want to go into which commits. How about I squash all commits together?

Reply:
This (introducing `DEFAULT`) should be the first commit. This commit is not very isolated (doesn't only change parser part) as it invalidates changes introduced previously. In previous commits new methods are introduced which are then changed by this commit. Such methods should be added once (in final form) within PR. We avoid moving back and forth for new code within single PR.